src/modules/Game.class.js

```diff
@@ -6,6 +6,25 @@
  * Feel free to add more props and methods if needed.
  */
 class Game {
+  static get deafultSize() {
+    return 4;
+  }
+
+  static get deafultInitialState() {
```

Comment:
Similarly, 'deafultInitialState' contains a typo. It should be 'defaultInitialState'.